Lib/pprint.py

```diff
@@ -342,6 +342,32 @@ def _pprint_mappingproxy(self, object, stream, indent, allowance, context, level
 
     _dispatch[_types.MappingProxyType.__repr__] = _pprint_mappingproxy
 
+    def _pprint_simplenamespace(self, object, stream, indent, allowance, context, level):
+        write = stream.write
+        cls_name = object.__class__.__name__
+        if cls_name == "SimpleNamespace":
+            # the SimpleNamespace repr is "namespace"
+            cls_name = "namespace"
+        indent += len(cls_name) + 1
+        delimnl = ',\n' + ' ' * indent
+        items = object.__dict__.items()
+        last_index = len(items) - 1
+
+        stream.write(cls_name + '(')
```

Comment:
Either you get rid of `write = stream.write` above or you use it here.

Reply:
Ah, you caught that too!! :p

Reply:
Agree, that it is worth to use just `write` here if we already use it in the loop.